### PR TITLE
Heroku prod

### PR DESCRIPTION
**Pull request by deadlybutter**
This PR makes two quick fixes so

1) Heroku prod deploy works
2) Users can back out of Northstar auth

### CODE REVIEW

File: bootstrap/setup.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ## Used by Heroku review apps to setup the application
 
 php artisan migrate --seed --force
+npm install --only=dev
```
Comment (weerd) [inline bootstrap/setup.sh]: ?

Comment (deadlybutter) [inline bootstrap/setup.sh]: on prod the NODE_ENV is production, and moderniz & whatnot isnt installed which breaks the build. so this just makes sure that all gets installed

Comment (deadlybutter) [inline bootstrap/setup.sh]: a npm install is run by the Heroku buildpack already, so we already have the normal dependencies at this point

Comment (weerd) [inline bootstrap/setup.sh]: 🤔

**Pull request merged**